control/joy_controller/config/joy_controller.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    joy_type: DS4
+    joy_type: $(var joy_type)
```

Comment:
How is this `$(var joy_type)` resolved?

This non-resolved variable causes the CI for this to fail:
https://github.com/autowarefoundation/autoware.universe/actions/runs/7981506848/job/21793322599#step:3:45

```
🔍 Validating: control/joy_controller/config/joy_controller.param.yaml 🆚 control/joy_controller/schema/joy_controller.schema.json ❌ Failed
Schema validation errors were encountered.
  control/joy_controller/config/joy_controller.param.yaml::$./**.ros__parameters.joy_type: '$(var joy_type)' is not one of ['P65', 'DS4', 'G29', 'XBOX']
```

Is there a specific reason to keep this as `$(var joy_type)` or could you revert this back to a default to comply with the schema file?

@yuki-takagi-66 @kaspermeck-arm

Reply:
Slightly (but not much) related to:
- https://github.com/autowarefoundation/autoware-github-actions/pull/285

Reply:
Fixing this in:
- https://github.com/autowarefoundation/autoware.universe/pull/6474

Reply:
@xmfcx 

Just looking into this, find
* https://github.com/autowarefoundation/autoware-github-actions/issues/232#issuecomment-1857640095

regarding how to use environmental variables. Is `joy_type` really an environmental variable like `$HOME`?

@oguzkaganozt @ambroise-arm

Reply:
@xmfcx 

Thanks for creating the PR! Just added a minor comment.